### DISCUSSION TITLE
Permissions lost

### DISCUSSION THREAD

**Discussion by ghost** [pulsar-edit/.github#40] [General]
I don`t have triage access, I have read, because of the following:

![image](https://user-images.githubusercontent.com/108250871/193469459-ad90a343-ede1-4857-acdc-6c4a64ed08ad.png)

And, it was **without notice.**

What happened? What is a reason for this?

**Comment by kaosine** [comment]
Hi, I think we originally set the default permissions on roles a little too high so we have reverted the role permissions back to what we originally envisioned. We still want you as part of this organisation but we feel that from some recent actions we need to make sure you are happy with and understand the permissions of the various roles, as well as the effect they can have on the project. Please keep adding your contributions and we will review your role assignment once we are all comfortable with the rapidly developing workflow.

**Reply by ghost** [reply]
What actions did I do bad? How did I create an impact?

Please answer these questions.

**Reply by kaosine** [reply]
- Premature approvals
- short messages that don't provide clarity on what you mean, especially in prs and occasionally in replies on a lot of other things
- when you have provided longer messages voicing a idea, it often comes off as demanding and combative instead of letting us go through with the democratic process on things that we'd established.

Plus, this is how the perms should've been before, with you moving up to triage eventually. It would be nice if w/out paying for enterprise(I think that's what it is in the settings) to more gradually give permissions. We're limited to the 5 roles, and we're trying early on to establish the kind of community we want, and these fixes to the perms should've happened when originally made. I will move you up again once I feel comfortable doing so. If you have any other questions, they can be directed at @confused-Techie when they're back (since I think they are now out of town for a couple of weeks). Or any other admin in the org.

**Reply by kaosine** [reply]
Also, don't take things out on us in private. If you have an issue with us, be up front about it and not behind our backs.  It was only ever said you had things to improve upon before gaining it back. Especially since this is how we meant for our perms to work with working up to more. Since I would've added you directly to this originally and moved you up based on how well you did here. I'll give you another chance to gain it back but you can't take things out on us in private for no reason. But I will be locking this thread.
